pyproject.toml

```diff
@@ -23,6 +23,7 @@ sentence-transformers = "^3.0.1"
 diskcache = "^5.6.3"
 openai = "^1.38.0"
 tenacity = "<9.0.0"
+groq="^0.11.0"
```

Comment:
We intentionally didn't add groq and anthropic as direct dependencies in poetry as it is an expectation on the user's end to have those installed in their environment if they are using Grok or Anthropic LLM clients. We don't want to have all the users install those.